File: AlarmManagerTp/app/src/main/java/com/example/alarmmanagertp/MainActivity.java
package com.example.alarmmanagertp;

import androidx.appcompat.app.AppCompatActivity;

import com.example.alarmmanagertp.FlashMemoryReporterScheduler;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        new FlashMemoryWearLevelAlarmReceiver();
        new FlashMemoryReporterScheduler(this);

    }
}

File: AlarmManager/app/src/main/java/com/example/alarmmanager/MainActivity.kt
package com.example.alarmmanager

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.alarmmanagertp.FlashMemoryReporterScheduler
import com.example.alarmmanagertp.FlashMemoryWearLevelAlarmReceiver

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        FlashMemoryWearLevelAlarmReceiver()
        FlashMemoryReporterScheduler(this)
    }
}
